.neuro/live-s3.yml

```diff
@@ -2,53 +2,6 @@ kind: live
 title: demo-oss-dogs
 id: mlops_demo_oss_dogs
 
-defaults:
-  preset: cpu-medium
-  life_span: 1d
-  env:
-    # Those env vars will be assigned to every job in this flow
-    PROJECT_GIT_REPO: git@github.com:neuro-inc/mlops-demo-oss-dogs.git
-    PROJECT_GIT_BRANCH: ${{ git.branch }}
-    BUCKET_NAME: mlops-oss-demo-dogs
-    CREDENTIALS: mlops-demo-oss-dogs-credentials
-    MLFLOW_TRACKING_URI: https://demo-oss-dogs-mlflow-server--alexeynaiden.jobs.default.org.neu.ro
-    PACHY_URI: pachyderm.green-hgx-2.org.neu.ro:30650
-    PACHY_REPO: dogs-demo
-
-volumes:
-  remote_dataset:
-    remote: storage:/$[[ project.owner ]]/$[[ flow.flow_id ]]/dataset
-    mount: /dataset
-  src:
-    remote: storage:/$[[ project.owner ]]/$[[ flow.flow_id ]]/src
-    mount: /usr/project/src
-    local: src
-  config:
-    remote: storage:/$[[ project.owner ]]/$[[ flow.flow_id ]]/config
-    mount: /usr/project/config
-    local: config
-  notebooks:
-    remote: storage:/$[[ project.owner ]]/$[[ flow.flow_id ]]/notebooks
-    mount: /usr/project/notebooks
-    local: notebooks
-
-images:
-  train:
-    ref: image:/$[[ project.owner ]]/$[[ flow.flow_id ]]/train:21.5.13
-    dockerfile: $[[ flow.workspace ]]/Dockerfile
-    context: $[[ flow.workspace ]]/
-    build_preset: cpu-medium
-  seldon:
-    ref: image:/$[[ project.owner ]]/$[[ flow.flow_id ]]/seldon:20.12.16
-    dockerfile: $[[ flow.workspace ]]/seldon/seldon.Dockerfile
-    context: $[[ flow.workspace ]]/
-    build_preset: cpu-medium
-  label_studio:
-    ref: image:/$[[ project.owner ]]/$[[ flow.flow_id ]]/label_studio:21.1.18
-    dockerfile: $[[ flow.workspace ]]/label_studio/label_studio.Dockerfile
-    context: $[[ flow.workspace ]]/label_studio
-    build_preset: cpu-medium
-
 jobs:
   prepare_remote_dataset:
```

Comment:
Let's put this job def from here and pachy live to project.yaml as a job.

Same for deploy_inference_platform, locust, filebrowser, postgres.